requests/exceptions.py

```diff
@@ -104,9 +104,18 @@ class StreamConsumedError(RequestException, TypeError):
 class RetryError(RequestException):
     """Custom retries logic failed"""
 
+
 class UnrewindableBodyError(RequestException):
     """Requests encountered an error when trying to rewind a body"""
 
+
+class ConflictingHeaderError(RequestException):
```

Comment:
Yeah, might be better to use that new exception, or possibly to make this a child exception of `InvalidHeader`.